src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -24,25 +25,52 @@
       :sub-label           (i18n/label :t/add-address-description)
       :add-divider?        true}]]])
 
-(def account-cards
-  [{:name                "Account 1"
-    :balance             "€0.00"
-    :percentage-value    "€0.00"
-    :customization-color :blue
-    :type                :empty
-    :emoji               "🍑"
-    :on-press            #(rf/dispatch [:navigate-to :wallet-accounts])}
-   {:customization-color :blue
-    :on-press            #(rf/dispatch
-                           [:show-bottom-sheet {:content new-account}])
-    :type                :add-account}])
+(def add-account-placeholder
+  {:customization-color :blue
+   :on-press            #(rf/dispatch [:show-bottom-sheet {:content new-account}])
+   :type                :add-account})
 
 (def tabs-data
   [{:id :assets :label (i18n/label :t/assets) :accessibility-label :assets-tab}
    {:id :collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}
    {:id :activity :label (i18n/label :t/activity) :accessibility-label :activity-tab}])
 
-(defn view
+(defn calculate-raw-balance
+  [rawBalance decimals]
+  (/ (js/parseInt rawBalance) (Math/pow 10 (js/parseInt decimals))))
+
+(defn calculate-balance
+  [{:keys [address]}]
+  (let [tokens       (rf/sub [:wallet-2/tokens])
+        token        (get tokens (keyword (string/lower-case address)))
+        total-values (atom 0)]
+    (doseq [item token]
+      (let [total-value-per-token (atom 0)]
+        (doseq [balances (vals (:balancesPerChain item))]
+          (reset! total-value-per-token (+ (calculate-raw-balance (:rawBalance balances)
+                                                                  (:decimals item))
+                                           @total-value-per-token)))
+        (reset! total-values (+ (* @total-value-per-token
+                                   (get-in item [:marketValuesPerCurrency :USD :price]))
+                                @total-values))))
```

Comment:
Can we convert keys to kebab-case?

Reply:
No, they're coming from the back end

Reply:
I think we convert them in the subs still thought, right @briansztamfater ? 🤔 
perhaps you can provide an example for this?

Reply:
This is the new function I wrote using `reduce` function:

`(defn calculate-balance
  [{:keys [address]}]
  (let [tokens       (rf/sub [:wallet-2/tokens])
        token        (get tokens (keyword (string/lower-case address)))
        result       (reduce (fn [acc item]
                               (let [total-value-per-token (reduce (fn [ac balances]
                                                                     (+ (calculate-raw-balance (:rawBalance balances)
                                                                                               (:decimals item))
                                                                        ac))
                                                                   0
                                                                   (vals (:balancesPerChain item)))
                                     total-values (* total-value-per-token
                                                     (get-in item [:marketValuesPerCurrency :USD :price]))]
                                 (+ acc total-values)))
                             0
                             token)] 
    (.toFixed result 2)))`